src/libs/ReportActionsUtils.ts

```diff
@@ -592,6 +618,25 @@ function getSortedReportActionsForDisplay(reportActions: ReportActions | null |
     return getSortedReportActions(baseURLAdjustedReportActions, true);
 }
 
+/**
+ * This method returns a combined array of report actions from a parent report and child transaction thread report that
+ * are ready for display in the ReportActionView.
+ */
+function getCombinedReportActionsForDisplay(reportActions: ReportAction[], transactionThreadReportActions: ReportAction[]): ReportAction[] {
+    if (_.isEmpty(transactionThreadReportActions)) {
+        return reportActions;
+    }
+
+    // Filter out the created action from the transaction thread report actions, since we already have the parent report's created action
+    const filteredTransactionThreadReportActions = transactionThreadReportActions?.filter((action) => action.actionName !== CONST.REPORT.ACTIONS.TYPE.CREATED);
+
+    // Sort the combined list of parent report actions and transaction thread report actions
+    const sortedReportActions = getSortedReportActions([...reportActions, ...filteredTransactionThreadReportActions], true);
+
+    // Filter out "created" IOU report actions because we don't want to show any preview actions for one transaction reports
+    return sortedReportActions.filter((action) => ((action as OriginalMessageIOU).originalMessage?.type ?? '') !== CONST.IOU.REPORT_ACTION_TYPE.CREATE);
```

Comment:
We intentionally don't filter the combined reportActions and filteredTransactionThreadReport actions first because we only want to remove the `created` actions from the transaction thread report actions here. 

We could do it this way if you find that clearer: 
```javascript
// Filter out the created action from the transaction thread report actions, since we already have the parent report's created action in `reportActions`
const filteredTransactionThreadReportActions = transactionThreadReportActions?.filter((action) => action.actionName !== CONST.REPORT.ACTIONS.TYPE.CREATED);

// Filter out "created" IOU report actions because we don't want to show any preview actions for one transaction reports    
const filteredReportActions = [...reportActions, ...filteredTransactionThreadReportActions].filter((action) => ((action as OriginalMessageIOU).originalMessage?.type ?? '') !== CONST.IOU.REPORT_ACTION_TYPE.CREATE);

return  getSortedReportActions(filteredReportActions, true);
```